lib/constantcontact/services/email_marketing_service.rb

```diff
@@ -51,6 +51,21 @@ def get_campaign(campaign_id)
           Components::Campaign.create(JSON.parse(response.body))
         end
 
+        # Get campaign email content for a specific campaign
+        # @param [Integer] campaign_id - Valid campaign id
+        # @return [Campaign]
+        def get_campaign_preview(campaign_id)
+          url = Util::Config.get('endpoints.base_url') +
+                sprintf(Util::Config.get('endpoints.campaign_preview'), campaign_id)
+          url = build_url(url)
+          response = RestClient.get(url, get_headers())
+          data = JSON.parse(response.body)
+          campaign = Components::Campaign.create(data)
```

Comment:
There's a few different ways we could tackle this...since the endpoint is unique, and doesn't technically map to any fields in the Campaign object, it might make sense to create a new object type of CampaignPreview, and allow that as an optional object inside of the Campaign object. Interestingly enough, I don't think campaign preview has been addressed in any of the current sdk's, so it would probably be best to tackle it the same way everywhere.